src/IsotonicFPOP.cpp

```diff
@@ -1,23 +1,24 @@
 /* -*- compile-command: "R CMD INSTALL .." -*- */
-  
-  #include <vector>
-  #include <stdio.h>
-  #include "funPieceListLog.h"
-  #include <math.h>
-  
-  void IsotonicFPOP
-(int *data_vec, double *weight_vec, int data_count,
```

Comment:
This is a fair comment, and I had left it for a while, but in an effort to find a bug I tried to simplify as much as possible to try to get everything to work. I can add this back in once I'm confident everything is working well! Thanks!